research/deep_speech/data/featurizer.py

```diff
@@ -23,10 +23,14 @@
 
 def compute_spectrogram_feature(samples, sample_rate, stride_ms=10.0,
                                 window_ms=20.0, max_freq=None, eps=1e-14):
-  """Compute the spectrograms for the input samples(waveforms)."""
+  """Compute the spectrograms for the input samples(waveforms).
+
+  More about spectrogram computation, please refer to:
+  https://en.wikipedia.org/wiki/Short-time_Fourier_transform.
+  """
   if max_freq is None:
-    max_freq = sample_rate / 2
-  if max_freq > sample_rate / 2:
+    max_freq = sample_rate // 2
```

Comment:
Do we need max_freq to be integer here?

Reply:
I think so.. frequency should be treated as an integer.

Reply:
But if you check line 61 of variable freqs, it's a floating number. Could you confirm this part? I think we need to make it concise.